src/main/java/ch/teachu/teachuapi/exam/ExamController.java

```diff
@@ -0,0 +1,55 @@
+package ch.teachu.teachuapi.exam;
+
+import ch.teachu.teachuapi.dtos.MessageResponse;
+import ch.teachu.teachuapi.exam.dto.ChangeExamRequest;
+import ch.teachu.teachuapi.exam.dto.CreateExamRequest;
+import ch.teachu.teachuapi.exam.dto.ExamHasGradesResponse;
+import ch.teachu.teachuapi.exam.dto.SemestersExamsResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@Tag(name = "Exam")
+@RequestMapping("/exam")
+@RestController
+public class ExamController {
+
+    private final ExamService examService;
+
+    public ExamController(ExamService examService) {
+        this.examService = examService;
+    }
+
+    @Operation(summary = "Load exams of teacher.")
+    @GetMapping
+    private ResponseEntity<SemestersExamsResponse> loadExams(@RequestHeader("auth") String auth) {
+        return examService.loadExams(auth);
+    }
+
+    @Operation(summary = "Create exam")
+    @PostMapping
+    private ResponseEntity<MessageResponse> createExam(@RequestHeader("auth") String auth, @RequestBody CreateExamRequest createExamRequest) {
+        return examService.createExam(auth, createExamRequest);
+    }
+
+    @Operation(summary = "Change exam")
+    @PutMapping
+    private ResponseEntity<MessageResponse> changeExam(@RequestHeader("auth") String auth, @RequestBody ChangeExamRequest changeExamRequest) {
+        return examService.changeExam(auth, changeExamRequest);
+    }
+
+    @Operation(summary = "Delete exam")
+    @DeleteMapping("/{examId}")
+    private ResponseEntity<MessageResponse> deleteExam(@RequestHeader("auth") String auth, @PathVariable UUID examId) {
+        return examService.deleteExam(auth, examId);
+    }
+
+    @Operation(summary = "Checks if the exam has grades. If it has then the exam is not deleteable.")
```

Comment:
We should check it in the delete endpoint and throw an exception. When it fails, the frontend can adapt to it. 
Multiple calls to do one thing should be avoided